api/src/services/graphqlInspector/graphqlInspector.ts

```diff
@@ -1,12 +1,42 @@
-import { coverage } from '@graphql-inspector/core'
+import { validate } from '@graphql-inspector/core'
 
 import { SyntaxError } from '@redwoodjs/graphql-server'
 
-// import { makeMergedSchema } from "@redwoodjs/graphql-server"
+import { logger } from 'src/lib/logger'
+import { getSchema } from 'src/util/graphqlSchema'
+
+/**
+ *
+ * @returns import {
+  diff,
+  validate,
```

Comment:
This looks strange :)  It started out as a doc comment, but then turned into something else?

Reply:
Removed. Better comment now.